lib/helpers.js

```diff
@@ -26,3 +26,28 @@ export function createBots(configFile) {
 
   return bots;
 }
+
+/**
+ * Returns occurances of a current channel member's name with `@${name}`
+ * @return {string}
+ */
+
+
+export function decorateBareUsernameWithAt(user, text) {
```

Comment:
Is the explicit URL logic necessary? What if we instead only replaced usernames that are preceded by whitespace or the start of the string, i.e. only check `^word\b` instead of checking `\bword\b` after splitting on spaces? I guess maybe you'd lose a few highlights, like when someone forgets to put a space after a comma, but maybe it'd be worth it nonetheless?